modules/seo-tools/jetpack-seo-utils.php

```diff
@@ -35,13 +35,13 @@ public static function is_enabled_jetpack_seo( $site_id = 0 ) {
 			return false;
 		}
 
-		if ( function_exists( 'has_blog_sticker' ) ) {
+		if ( function_exists( 'has_any_blog_stickers' ) ) {
 			// For WPCOM sites
 			if ( empty( $site_id ) ) {
 				$site_id = get_current_blog_id();
 			}
 
-			return has_blog_sticker( 'business-plan', $site_id );
+			return has_any_blog_stickers( [ 'business-plan', 'ecommerce-plan' ], $site_id );
```

Comment:
Could you edit this to not use short array syntax? We try to stay away from it in Jetpack since the plugin must still support PHP 5.2.
```suggestion
			return has_any_blog_stickers( array( 'business-plan', 'ecommerce-plan' ), $site_id );
```